packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -295,6 +295,29 @@ within the Menu upon opening.
 
 ### Disabled
 
+To disable Menu items, pass an array of `key`s or `id`s into the `disabledKeys` prop.
+If an item has both `key` and `id` attributes, the `key` attribute should be used for disabling.
```

Comment:
IMO I think it should be made clear that `disabledKey` expects an array of `key`'s. The bit about `id`'s is more about the array of `items` (rather than the `Item` in the render) where the `id` will be used as a substitute for the `key` if `key` is missing.

Perhaps something like this?
```suggestion
To disable Menu items, pass an array of keys into the `disabledKeys` prop.
If an item has both `key` and `id` attributes, the `key` attribute should be used for disabling.
See the [Content](#events) section for more detail on how an item's `key` may be determined.
```

Reply:
Sounds good!